batchwrite.go

```diff
@@ -74,21 +84,17 @@ func (bw *BatchWrite) Run() error {
 			return err
 		}
 
-		if res.UnprocessedItems == nil || len(res.UnprocessedItems) == 0 {
-			break
-		}
-		bw.ops = res.UnprocessedItems[bw.batch.table.Name()]
```

Comment:
I think deleting this here will ignore unprocessed items, potentially losing data.